packages/fluent-theme/src/components/sendbox/Toolbar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { type ButtonHTMLAttributes, type HTMLAttributes } from 'react';
+import cx from 'classnames';
+import { useStyles } from '../../styles';
+
+const styles = {
+  webchat__sendbox__toolbar: {
+    display: 'flex',
+    gap: '4px',
+    marginInlineStart: 'auto'
+  },
+
+  'webchat__sendbox__toolbar-button': {
+    display: 'flex',
+    appearance: 'none',
+    background: 'transparent',
+    alignItems: 'center',
+    justifyContent: 'center',
+    border: 'none',
+    cursor: 'pointer',
+    padding: '3px',
+    width: '32px',
+    aspectRatio: '1',
+    borderRadius: 'var(--borderRadiusSmall)',
+
+    '> svg': {
```

Comment:
Put class name in the `<svg>`.

Reply:
There may be custom icons there which will be specified by user